_pages/index.md

```diff
@@ -15,6 +15,20 @@ layout: home
 
 <section class="usa-section grid-container usa-prose" markdown="1">
 
+{%- capture alert_content -%}
```

Comment:
we could probably make one partial for this and use it on all these pages, and then include that everywhere? Like one partial specifically for this migration banner that itself includes the alert partial?

Reply:
yeah that would probably make even more sense. Let me take a stab at that